app/.babelrc

```diff
@@ -1,36 +1,44 @@
-{
-  "presets": [
-    "airbnb",
-    [
-      "@babel/preset-env",
-      {
-        "modules": false
-      }
-    ]
-  ],
-  "plugins": [
-    [
-      "babel-plugin-import",
-      {
-        "libraryName": "@material-ui/core",
-        "libraryDirectory": "esm",
-        "camel2DashComponentName": false
-      },
-      "core"
-    ],
-    [
-      "babel-plugin-import",
-      {
-        "libraryName": "@material-ui/icons",
-        "libraryDirectory": "esm",
-        "camel2DashComponentName": false
-      },
-      "icons"
-    ]
-  ],
-  "env": {
-    "production": {
-      "presets": ["minify"]
-    }
-  }
-}
+{
+  "presets": [
+    "airbnb",
+    [
+      "@babel/preset-env",
+      {
+        "modules": false
+      }
+    ]
+  ],
+  "plugins": [
+
+    ["@babel/plugin-transform-runtime",
+      {
+        "regenerator": true
+      }
+    ],

```

Comment:
What is this doing?

Reply:
It turns the regenerator runtime on, idk what it is but some things rely on it. It fixes an error